tachyon/zk/plonk/lookup/lookup_argument.h

```diff
@@ -69,6 +77,55 @@ class LookupArgument {
     return 2 + max_input_degree + max_table_degree;
   }
 
+  template <typename Domain, typename Evals, typename Commitment,
+            typename PCSTy>
```

Comment:
```suggestion
  template <typename PCSTy>
```

Reply:
Since the template arguments can be inferred by the function parameter, we don't need to apply this comment.